alfresco-search/src/main/java/org/alfresco/solr/tracker/MetadataTracker.java

```diff
@@ -591,7 +597,20 @@ protected Transactions getSomeTransactions(BoundedDeque<Transaction> txnsFound,
         Transactions transactions;
         // step forward in time until we find something or hit the time bound
         // max id unbounded
-        Long startTime = fromCommitTime == null ? Long.valueOf(0L) : fromCommitTime;
+        Long startTime = fromCommitTime == null  ? 0L : fromCommitTime;
+        log.debug(String.format("#### %s MetadataTracker getSomeTransactions start time: %d end: %d",
+                  this.coreName, 
+                  startTime,
+                  endTime));
+        if(startTime == 0)
+        {
+            return client.getTransactions(startTime,
+                                          null,
+                                          startTime + actualTimeStep, 
```

Comment:
There doesn't seem any point in the local variable actualTimeStep - it's always just set to timeStep I think.